api/src/pcapi/admin/custom_views/venue_view.py

```diff
@@ -414,10 +414,7 @@ def update_view(self) -> Response:
             # tags are used by other tools (eg. building playlists for the
             # home page) and waiting N minutes for the next indexing
             # cron tasks is painful.
-            if criteria or remove_other_tags:
-                search.reindex_venue_ids(venue_ids)
-            else:
-                search.async_index_venue_ids(venue_ids)
+            search.async_index_venue_ids(venue_ids)
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏Maintenant le commentaire ment !